### PR TITLE
Add abandoned checkout

### PR DESCRIPTION
**Pull request by senafu**
Add abandoned checkout

### CODE REVIEW

File: abandoned_checkout.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	"time"
+
+	"github.com/shopspring/decimal"
+)
+
+const abandonedCheckoutsBasePath = "checkouts"
+
+// AbandonedCheckoutService is an interface for interfacing with the abandonedCheckouts endpoints
+// of the Shopify API.
+// See: https://help.shopify.com/api/reference/online_store/checkout
```
Comment (oliver006) [inline abandoned_checkout.go:14]: This link doesn't work for me.
Maybe something like https://shopify.dev/docs/api/admin-rest/latest/resources/abandoned-checkouts is better?

**Review by oliver006 — COMMENTED** [review]
Thanks for the PR, looks good!
Just one minor comment about the link to the Shopify docs, it's not working.

File: abandoned_checkout.go (hunk 1 above)
Comment (senafu) [reply abandoned_checkout.go]: Thank you for your confirmation.
I fixed the link.
86f7696968fde81ceb9e321be8243b06aa18f8ce

Comment (senafu) [reply abandoned_checkout.go]: @oliver006 
Thank you for your confirmation.
I fixed the link.
https://github.com/bold-commerce/go-shopify/commit/86f7696968fde81ceb9e321be8243b06aa18f8ce

**Review by oliver006 — APPROVED** [review]
Looks good - thanks for the PR!

**Pull request merged**